src/protocols/aeternity/libs/AeSdkSuperhero.ts

```diff
@@ -29,15 +29,15 @@ type ISpendOptions = Omit<Parameters<typeof spend>[2], 'onAccount' | 'onNode'>
  * provides flexibility to manage the accounts the way wallet would like to handle
  */
 export class AeSdkSuperhero extends AeSdkWallet {
-  store: Store<any>;
+  account: AeAccountHdWallet;
 
-  constructor(store: Store<any>, options: any) {
+  constructor(options: any, nodeNetworkId: Ref<string | undefined>) {
     super(options);
-    this.store = store;
+    this.account = new AeAccountHdWallet(nodeNetworkId);
   }
 
   _resolveAccount() {
-    return new AccountSuperhero(this.store);
```

Comment:
Every composable that had some kind of data polling and was using the SDK was causing to fire this method again and again. So the `AccountSuperhero` was instantiated dozen times per minute. Now it's cached.